tests/Earthfile

```
@@ -610,6 +610,26 @@ wildcard-build-pwd:
     RUN cat earthly.output | acbgrep 'hello world 3'
     RUN cat earthly.output | acbgrep -v 'not-test run'
 
+wildcard-build-pwd-rel:
+    COPY wildcard-build subdir
+    COPY wildcard-build.earth subdir/Earthfile
+    DO +RUN_EARTHLY --earthfile=wildcard-build.earth --target=./subdir+wildcard-build-pwd
```

the `--earthfile=wildcard-build.earth` will copy that file into `./Earthfile` which is wrong, since you've already copied it into `./subdir`.
```suggestion
    DO +RUN_EARTHLY --target=./subdir+wildcard-build-pwd
```